yearzero/YZRoll.js

```diff
@@ -119,6 +119,24 @@ class YZRoll {
 			.length;
 	}
 
+	/**
+	 * The quantity of ones(banes) on base dice.
+	 * @type {number}
+	 * @readonly
+	 */
+	get baseBaneCount() {
```

Comment:
I just noticed that the getter `.baseBaneCount` does the same job as the getter `.attributeTrauma`.

It's repetitive code, but not a big deal.

Reply:
Fixed